common/predictive-text/build.sh

```diff
@@ -180,7 +199,8 @@ if (( fetch_deps )); then
   npm install --no-optional
 fi
 
-build || fail "Compilation failed."
+build || fail "Browser-oriented compilation failed."
+build_headless || fail "Headless compilation failed."
```

Comment:
If following my previous suggestion, then this need to be changed too:

```suggestion
build-browser || fail "Browser-oriented compilation failed."
build-headless || fail "Headless compilation failed."
```